shared_model/bindings/model_crypto.hpp

```diff
@@ -15,8 +15,8 @@
  * limitations under the License.
  */
 
-#ifndef IROHA_SIMPLE_CRYPTO_HPP
-#define IROHA_SIMPLE_CRYPTO_HPP
+#ifndef IROHA_SHARED_MODEL_MODEL_CRYPTO_HPP
```

Comment:
Why not `IROHA_SHARED_MODEL_CRYPTO_HPP`?

Reply:
I'm a bit suspicious about version with a single "MODEL" word for possible redefinings (there are none of them at the moment but I afraid of some of them possible in the future). Two "MODEL" words should be unique forever :)